Model/DutyCalculator.php

```diff
@@ -320,7 +319,11 @@ protected function prepareRequest($freightCharge, $shippingAddress)
                 if (!$itemData['countryOfOrigin']) {
                     $itemData['countryOfOrigin'] = $request['senderAddress']['country'];
                 }
-                $itemData['qualifiesForPreferentialTariffs']=true;
+                if ($this->reachHelper->getPrefTariffs() == 1 ) {
```

Comment:
does Magento restrict the Yes/No field to 1/0? or could we map those values to True/False and avoid this if statement